configs/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb.py

```diff
@@ -0,0 +1,135 @@
+# model settings
+model = dict(
+    type='Recognizer3D',
+    backbone=dict(
+        type='C3D',
+        pretrained=  # noqa: E251
+        'https://download.openmmlab.com/mmaction/recognition/c3d/c3d_sports1m_pretrain_20201016-dcc47ddc.pth',  # noqa: E501
```

Comment:
since you are ignoring E501, why not put the above two lines in one line?

Reply:
I put the two lines with reference to "configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb": line 7-8.
In fact, when I want to put them in one line, the IDE will automatically wrap them (it follows pep8 code style).

Reply:
And the `# noqa: E251` is used to pass CI

Reply:
which formatter are used? i think a good formatter should respect the `noqa` marks

Reply:
I used yapf, but I just switched to autopep8, and seems better.

Reply:
okay it is yapf's fault: https://github.com/google/yapf/issues/423

let's keep it in two lines for now since many devs use yafp.